userguide/intro.rst

```diff
@@ -215,14 +215,17 @@ This software has been added or updated:
 * rsync has been updated to
   `version 3.1.3 <https://download.samba.org/pub/rsync/src/rsync-3.1.3-NEWS>`__.
 
+* rclone has been updated to
+  `version 1.43 <https://rclone.org/changelog/#v1-43-2018-09-01>`__.
```

Comment:
Its actually 1.44, ticket ticket is out of data

Reply:
Thanks, updated.